pkg/slo-controller/noderesource/plugins/batchresource/plugin.go

```diff
@@ -133,6 +133,14 @@ func (p *Plugin) Calculate(strategy *extension.ColocationStrategy, node *corev1.
 	nodeUsage := getNodeMetricUsage(nodeMetric.Status.NodeMetric)
 	systemUsed := quotav1.Max(quotav1.Subtract(nodeUsage, podAllUsed), util.NewZeroResourceList())
 
+	// System.Used = max(System.Used, Node.Anno.Reserved)
+	nodeReserved := util.GetNodeReservationFromAnnotation(node.Annotations)
```

Comment:
To ensure the calculation is correct, please add a UT case for the node resource reservation later.

Reply:
done